javascript/src/main/components/Channels/ChannelLinksTable.js

```diff
@@ -0,0 +1,22 @@
+// import React from "react";
```

Comment:
This file looks like it is from another feature. Take this file out from your branch